source/common/http/utility.h

```diff
@@ -231,6 +231,12 @@ struct GetLastAddressFromXffInfo {
 GetLastAddressFromXffInfo getLastAddressFromXFF(const Http::HeaderMap& request_headers,
                                                 uint32_t num_to_skip = 0);
 
+/**
+ * Remove any headers nominated by the Connection header
+ * Sanitize the TE header if it contains unsupported values
+ */
+void sanitizeConnectionHeader(spdlog::logger& logger, Http::HeaderMap& headers);
```

Comment:
Drive by: I don't think it's great to pass loggers around, we should ideally not be using low-level `spdy` types on internal utilities. Just use a `ENVOY_LOG_MISC` in utils.

Reply:
Thanks, this makes sense.  I didn't see that macro initially.

Reply:
+1, thanks